code/game/machinery/autolathe.dm

```diff
@@ -22,74 +22,187 @@
 	var/hack_wire
 	var/disable_wire
 	var/shock_wire
-
-	var/busy = FALSE
 	var/prod_coeff = 1
-
-	var/datum/design/being_built
 	var/datum/techweb/stored_research
-	var/list/datum/design/matching_designs
-	var/selected_category
 	var/screen = 1
 	var/base_price = 25
 	var/hacked_price = 50
-
-	var/list/categories = list(
-							"Tools",
-							"Electronics",
-							"Construction",
-							"T-Comm",
-							"Security",
-							"Machinery",
-							"Medical",
-							"Misc",
-							"Dinnerware",
-							"Imported"
-							)
+	var/datum/research/files
+	var/search
+	var/datum/material_container/materials
+	var/queue_max_len = 12
+	var/processing_queue = 0
+	var/datum/design/item_beingbuilt
+	var/datum/design/request
+	var/list/being_built = list()
+	var/list/autoqueue = list()
+	var/list/uidata = list()
```

Comment:
I'm pretty sure this var is never actually used - below there's a *local* var named uidata but that's not this var.